src/test/javascript/spec/component/standardized-competencies/admin-import-standardized-competencies.spec.ts

```diff
@@ -15,14 +15,28 @@ import { HttpResponse } from '@angular/common/http';
 import { of } from 'rxjs';
 import { KnowledgeAreasForImportDTO } from 'app/entities/competency/standardized-competency.model';
 import { StandardizedCompetencyDetailComponent } from 'app/shared/standardized-competencies/standardized-competency-detail.component';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { ArtemisSharedModule } from '../../../../../main/webapp/app/shared/shared.module';
+import { ArtemisSharedComponentModule } from '../../../../../main/webapp/app/shared/components/shared-component.module';
+import { KnowledgeAreaTreeComponent } from '../../../../../main/webapp/app/shared/standardized-competencies/knowledge-area-tree.component';
+import { ArtemisMarkdownModule } from '../../../../../main/webapp/app/shared/markdown.module';
 
 describe('AdminImportStandardizedCompetenciesComponent', () => {
     let componentFixture: ComponentFixture<AdminImportStandardizedCompetenciesComponent>;
     let component: AdminImportStandardizedCompetenciesComponent;
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [ArtemisTestModule, NgbCollapseMocksModule],
+            imports: [
```

Comment:
Consider coderabbits suggestion. I don't think we need all of these imports and declarations.